src/main/cli/cli.c

```diff
@@ -5225,6 +5263,12 @@ static void printConfig(char *cmdline, bool doDiff)
 #ifdef USE_OSD
         cliPrintHashLine("display_name");
         printDisplayName(dumpMask, &pilotConfig_Copy);
+
+        cliPrintHashLine("rateprofile_name");
```

Comment:
This is in the wrong place - it should be listed as part of the PID / rate profile, so that in a `diff all` all of the names are printed, to make backup / restore work as intended.

Reply:
Good point.